### PR TITLE
fix: add levels.fyi field to companies database 🚀

### LINKED ISSUE #319: Add 'levels_fyi' field to database 📖
### Description

The company name used in the levels_fyi URL of a page is not always clear (i.e. JPMorgan is `jpmorgan-chase`), so we need to store the levels_fyi URL name in a database field. 

### Acceptance Criteria

Add a field named `levels_fyi` to the `companies` by implementing a database migration. 

### Additional Context

See documentation for creating a database migration [here](how-to-implement-a-database-migration.md), and see various examples of migrations in the [migrations](https://github.com/colorstackorg/oyster/tree/main/packages/db/src/migrations) folder! This is where the new migration will be stored.

### PR DESCRIPTION
**Pull request by lianarosa**
## Description ✏️
 
- Migration: `20240622005304_add_levels_fyi.ts`
- Inserts levels_fyi column into companies table.

Closes #319

## Type of Change 🐞

- [ ] Feature - A non-breaking change which adds functionality.
- [X] Fix - A non-breaking change which fixes an issue.
- [ ] Refactor - A change that neither fixes a bug nor adds a feature.
- [ ] Documentation - A change only to in-code or markdown documentation.
- [ ] Tests - A change that adds missing unit/integration tests.
- [ ] Chore - A change that is likely none of the above.

## Checklist ✅

- [X] I have done a self-review of my code.
- [X] I have manually tested my code (if applicable).
- [ ] I have added/updated any relevant documentation (if applicable).

### CODE REVIEW

File: packages/db/src/migrations/20240622005304_add_levels_fyi.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    .addColumn('levels_fyi', 'text')
+    .execute();
+}
+
+export async function down(db: Kysely<any>) {
+  await db.schema
+    .alterTable('companies')
+    .dropColumn('levels_fyi')
+    .execute();
+}
```
Comment (tomas-salgado) [inline packages/db/src/migrations/20240622005304_add_levels_fyi.ts:15]: It looks like you might be missing the VSCode extension [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode). You can download this within VSCode and then it should fix the formatting of this file (which is what's giving the error) upon saving

**Review by tomas-salgado — DISMISSED** [review]
Nice work, @lianarosa! It looks like there's just a formatting issue to be addressed but otherwise it all looks good!

**Comment by wflore19** [comment]
> Nice work, @lianarosa! It looks like there's just a formatting issue to be addressed but otherwise it all looks good!

I'm getting this same format here [lianarosa:addlevelsfyi](https://github.com/colorstackorg/oyster/pull/330/commits/12464abd3870d3a910a1b95f2430dac9549e44b8) with prettier installed, but the bot asked @lianarosa to fix it with prettier. Is that not the correct format w/ prettier? And then she made the change [lianarosa:addlevelsfyi](https://github.com/colorstackorg/oyster/pull/330/commits/beb3c7bb40e3415d66db8fa77c8778f620c1b15e) and again, the bot asked her to fix it with prettier.

My result with prettier installed:
image

**Comment by tomas-salgado** [comment]
> > Nice work, @lianarosa! It looks like there's just a formatting issue to be addressed but otherwise it all looks good!
> 
> I'm getting this same format here [lianarosa:addlevelsfyi](https://github.com/colorstackorg/oyster/pull/330/commits/12464abd3870d3a910a1b95f2430dac9549e44b8) with prettier installed, but the bot asked @lianarosa to fix it with prettier. Is that not the correct format w/ prettier? And then she made the change [lianarosa:addlevelsfyi](https://github.com/colorstackorg/oyster/pull/330/commits/beb3c7bb40e3415d66db8fa77c8778f620c1b15e) and again, the bot asked her to fix it with prettier.
> 
> My 20240622005304_add_levels_fyi.ts file with prettier installed:
> 
> ```
> import { type Kysely } from 'kysely';
> 
> export async function up(db: Kysely<any>) {
>   await db.schema
>     .alterTable('companies')
>     .addColumn('levels_fyi', 'text')
>     .execute();
> }
> 
> export async function down(db: Kysely<any>) {
>   await db.schema.alterTable('companies').dropColumn('levels_fyi').execute();
> }
> ```

I believe the current state of this PR does not have the formatting that you gave, and I think this is why the test is failing. Right now it looks like the formatting of the `down` function is not all in one line.

**Review by ramiAbdou — APPROVED** [review]

**Review by ramiAbdou — APPROVED** [review]

**Pull request merged**